<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CdChecker</class>
 <widget class="QWidget" name="CdChecker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="topWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="albumTitleLabel">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>CD</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="buttonWidget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="enqueueAllButton">
           <property name="text">
            <string>Enqueue All</string>
           </property>
           <property name="icon">
            <iconset theme="view-media-playlist">
             <normaloff>../MacIntegration/cd</normaloff>../MacIntegration/cd</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="playAllButton">
           <property name="text">
            <string>Play All</string>
           </property>
           <property name="icon">
            <iconset theme="media-playback-start">
             <normaloff>../MacIntegration/cd</normaloff>../MacIntegration/cd</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="shuffleAllButton">
           <property name="text">
            <string>Shuffle All</string>
           </property>
           <property name="icon">
            <iconset theme="media-playlist-shuffle">
             <normaloff>../MacIntegration/cd</normaloff>../MacIntegration/cd</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="importCdButton">
           <property name="text">
            <string>Import</string>
           </property>
           <property name="icon">
            <iconset theme="media-cd-import">
             <normaloff>../MacIntegration/cd</normaloff>../MacIntegration/cd</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ejectButton">
           <property name="text">
            <string>Eject</string>
           </property>
           <property name="icon">
            <iconset theme="media-eject">
             <normaloff>../MacIntegration/cd</normaloff>../MacIntegration/cd</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>1</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="tracksWidget">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="musicBrainzWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
